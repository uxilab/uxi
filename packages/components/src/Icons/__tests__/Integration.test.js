import React from 'react';
import Integration from '../Integration';

describe('Icon : <Integration />', () => {
  it('match snapshot', () => {
    expect(shallow(<Integration />)).toMatchSnapshot();
  });
});

