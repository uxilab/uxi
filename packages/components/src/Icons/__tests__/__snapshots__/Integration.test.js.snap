// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Integration /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M12.94,6.82a3,3,0,0,0-.59.06l-.1,0V5.24a1.49,1.49,0,0,0-1.49-1.49H9.1l0-.1a3,3,0,0,0,.06-.59,3.06,3.06,0,0,0-6.12,0,3,3,0,0,0,.06.59l0,.1H1.49A1.49,1.49,0,0,0,0,5.24V7.78a.6.6,0,0,0,.91.51,1.84,1.84,0,0,1,1-.28,1.87,1.87,0,0,1,0,3.74,1.84,1.84,0,0,1-1-.28A.6.6,0,0,0,0,12v2.54A1.49,1.49,0,0,0,1.49,16h9.27a1.49,1.49,0,0,0,1.49-1.49V12.86l.1,0a3,3,0,0,0,.59.06,3.06,3.06,0,1,0,0-6.12Zm0,4.93a1.84,1.84,0,0,1-1-.28.6.6,0,0,0-.91.51v2.54a.3.3,0,0,1-.3.3H1.49a.3.3,0,0,1-.3-.3V12.86l.1,0a3,3,0,0,0,.59.06,3.06,3.06,0,1,0,0-6.12,3,3,0,0,0-.59.06l-.1,0V5.24a.3.3,0,0,1,.3-.3H4A.6.6,0,0,0,4.54,4a1.83,1.83,0,0,1-.28-1A1.87,1.87,0,0,1,8,3.06a1.84,1.84,0,0,1-.28,1,.6.6,0,0,0,.51.91h2.54a.3.3,0,0,1,.3.3V7.78a.6.6,0,0,0,.91.51,1.84,1.84,0,0,1,1-.28,1.87,1.87,0,1,1,0,3.74Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
